src/tests/specs/ink/Misc.spec.ts

```diff
@@ -196,8 +196,7 @@ C C C
   // TestReturnTextWarning
   it("tests return text warning", () => {
     compileStoryWithoutRuntime("return_text_warning");
-
-    expect(context.warningMessages.length).toBe(1);
+    expect(context.warningMessages.length).toBe(2); // inklecate returns 2 warnings
```

Comment:
Inklecate and inky actually return 2 errors, the original test only verifies there's at least (and not _exactly_) 1 warning